spec/project.schema.json

```diff
@@ -6,20 +6,25 @@
       "type": "string"
     },
     "samples": {
-      "type": "array",
-      "items": {
-        "type": "object",
-        "properties": {
-          "hash": {
-            "type": "string"
+      "type": "object",
+      "patternProperties": {
+        "^[a-zA-Z0-9]*$": {
+          "type": "object",
+          "properties": {
+            "hash": {
+              "type": "string"
+            },
+            "cluster": {
+              "type": ["integer", "string"]
+            },
```

Comment:
So cluster is no longer required as it will be undefined if sample failed. But can we really return either an integer or a string now - does it return a number if it can't find a cluster name?

Reply:
im actually not sure? ive never seen a number be returned... so il just update to just string